src/jsMain/kotlin/bigComponents/Bookmark.kt

```diff
@@ -267,10 +273,10 @@ val BookmarkPage = functionalComponent<RProps> { _ ->
                             width = 90.pct
                             height = 60.px
                             if (category != "lowerSection") {
-                                backgroundColor = Color.white
+                                backgroundColor = Color.floralWhite
```

Comment:
This is repeated quite a bit.  Please make this into a component that you can reuse